src/osmpbf.rs

```diff
@@ -193,22 +165,21 @@ pub fn read_block<F: Read + Seek, T: prost::Message + Default>(
 }
 
 fn blob_type_from_blob_info(
-    blob_buf: &mut Vec<u8>,
```

Comment:
Keep this one for best perf?

Reply:
Can't, since it used inside of the par iter now.